app.py

```diff
@@ -477,7 +478,22 @@ def get_latest_github_commits():
         # author (github name), date, and URL to the changes
         commit_name = commitsArr[repo_num].commit.message.split("\n")[0]
         commit_author = commitsArr[repo_num].commit.author.name
-        commit_date = str(commitsArr[repo_num].commit.author.date)
+        # raw date input from pygithub
+        raw_date = str(commitsArr[repo_num].commit.author.date)
+        # convert it to a datetime object, with pytz specifying this is UTC
+        datetime_object = datetime.datetime(
+            int(raw_date[0:4]),
+            int(raw_date[5:7]),
+            int(raw_date[8:10]),
+            int(raw_date[11:13]),
+            int(raw_date[14:16]),
+            int(raw_date[17:19]),
+            tzinfo=pytz.utc,
+        )
+        # use pytz again to change it to LA time
+        commit_date = datetime_object.astimezone(
```

Comment:
Rather than force PST here, I'd move this logic to JS where you can convert to the user's local time using a Date() object.